trunk/SUAVE/Components/Energy/Converters/Internal_Combustion_Engine.py

```diff
@@ -0,0 +1,102 @@
+# Internal_Combustion_Engine.py
+#
+# Created:  Aug, 2016: D. Bianchi
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+from SUAVE.Core import Data, Units
+
+# package imports
+import numpy as np
+from SUAVE.Components.Energy.Energy_Component import Energy_Component
+
+# ----------------------------------------------------------------------
+#  Internal Combustion Engine Class
+# ----------------------------------------------------------------------
+
+class Internal_Combustion_Engine(Energy_Component):
+
```

Comment:
This file is missing docstrings and doxygen info.

Reply:
Doxygen info added

Reply:
Docstring Added, Sorry I think I must have added a docstring to the file in the Networks folder but forgot the Converters folder